drivers/adc/Kconfig.tla2021

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2023 Caspar Friedrich <c.s.w.friedrich@gmail.com>
```

Comment:
Not blocking, but typically we don't see email addresses in the copyright. That's up to you though.

Reply:
true... but in order to not delay this any longer I would rather not change this.